ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -554,7 +554,7 @@ EventProductVersion,string,Optional,FileEvent,,,
 EventProductVersion,string,Optional,NetworkSession,,,
 EventProductVersion,string,Optional,ProcessEvent,,,
 EventProductVersion,string,Optional,RegistryEvent,,,
-EventProductVersion,string,Optional,UserManagement,,,
+EventProductVersion,string,Optional,UserManagement,Enumerated,SentinelOne,
```

Comment:
@jayeshprajapaticrest 
Please correct this

Reply:
pls double check this in other parsers

Reply:
Done